pkg/api/fnresult/v1alpha2/types.go

```diff
@@ -73,3 +73,20 @@ func NewResultList() *ResultList {
 		Items: []Result{},
 	}
 }
+
+type ResultItem struct {
```

Comment:
Add a brief note about this struct. It will also be good to highlight why we are duplicating this here instead of using `framework.ResultItem` from the kyaml.